scripts/tests.sh

```diff
@@ -36,6 +36,11 @@ source "${REPO_ROOT}/scripts/common.sh"
 WORKDIR=`mktemp -d`
 CMDLINE_PID=
 
+if [ "$1" = --no-smt ]
```

Comment:
I think this is wrong. Can't recall how to compare bash strings properly, but if I do I'll post it here

Reply:
Now removed. The right way is to export `SMT_FLAGS=--no-smt` before calling `tests.sh`.